queries.sql

```diff
@@ -92,4 +92,58 @@ SET weight_kg = weight_kg * -1;
 WHERE weight_kg < 0 ;
 SELECT * FROM animal;
 
-COMMIT;
\ No newline at end of file
+COMMIT;
+
+
+UPDATE animals
+SET owner_id = 1
+WHERE name ='Agumon';
+
+UPDATE animals
+SET owner_id = 2
+WHERE name IN ('Gabumon','Pikachu');
+
+UPDATE animals
+SET owner_id = 3
+WHERE name IN ('Devimon','Plantmon');
+
+UPDATE animals
+SET owner_id = 4
+WHERE name IN ('Charmander','Squirtle','Blossom');
+
+UPDATE animals
+SET owner_id = 5
+WHERE name IN ('Angemon','Boarmon');
+
+UPDATE animals
+SET species_id = 2
+WHERE name LIKE '%mon';
+
+UPDATE animals
+SET species_id = 1
+WHERE species_id IS NULL;
```

Comment:
Kindly move your UPDATE lines to your schema.sql file 📝 Everything that creates or **changes** the tables structure should be saved in the schema.sql file.

Reply:
I moved it to schema.sql file @mateo951. Thanks for your time reviewing my PR. It was a great help to me. I understand very well. I added owners and species in my terminal. I forgot to add it to my repository files.